src/external_secondaries/CMakeLists.txt

```diff
@@ -6,6 +6,14 @@ set(MAIN_ECU_INTERFACE_SRC ecuinterface.cc)
 add_executable(myecuinterface ${MAIN_ECU_INTERFACE_SRC} dummy.cc)
 target_link_libraries(myecuinterface ${Boost_LIBRARIES})
 
+if(NOT FLASHER_VERSION)
+    add_definitions(-DFLASHER_VERSION="1")
+endif(NOT FLASHER_VERSION)
```

Comment:
For now this can be hard coded--the API version information logically belongs with the code that calls the API and parses the results.  If we switch to version 2, then the parser will have to change too. Therefore I think it is better to put the version number in with the parser.

Reply:
OK